day_6/exercises/burrito.rb

```diff
@@ -11,9 +11,32 @@ def initialize(protein, base, toppings)
     @base     = base
     @toppings = toppings
   end
+
+  def add_topping(new_topping)
+    toppings.push(new_topping)
```

Comment:
why are you able to leverage this without the @ symbol prefixing toppings?

Reply:
I believe it has something to do with how I have the attr_reader, had I used attr_accessor I wouldn't need to use the @ on any of them. Is this correct?

Reply:
correct!